crates/oxc_cli/src/lint/runner.rs

```diff
@@ -29,6 +29,8 @@ pub struct LintRunner {
     options: LintOptions,
 
     linter: Arc<Linter>,
+
+    registered_rules: FxHashMap<RuleCategory, Vec<RuleEnum>>,
```

Comment:
Does it make sense to get the rules without storing it here? We have the `RULES` global

Reply:
Oh sorry, I misinterpreted the  requirements here, we just want to output all available rules right (and their associated counts) + the total? Should be fixed [here](https://github.com/Boshen/oxc/pull/290/commits/a947bd06c7c97fc8ffe7b3e698f21452675a70cb)